app/services/funnel/doc_auth/register_step_from_analytics_view_event.rb

```diff
@@ -4,7 +4,8 @@ class RegisterStepFromAnalyticsViewEvent
       ANALYTICS_EVENT_TO_DOC_AUTH_LOG_TOKEN = {
         Analytics::IDV_PHONE_RECORD_VISIT => :verify_phone,
         Analytics::IDV_REVIEW_VISIT => :encrypt,
-        Analytics::IDV_FINAL => :verified,
+        Analytics::IDV_PERSONAL_KEY_VISITED => :verified,
+        Analytics::IDV_FINAL => :verified, # deprecated
```

Comment:
Just confirming: Do we still need this? Specifically noting the "deprecation". Would this be removed after the next deploy, and do we need to be concerned about double-counting? Alternatively, do we need to make any changes here at all, or is it enough to keep this as it was previously, since we'll still be logging `IDV_FINAL`, and that event could still be associated with `:verified` ?